pkg/image/containerdregistry/resolver.go

```diff
@@ -85,14 +87,31 @@ func credential(cfg *configfile.ConfigFile) func(string) (string, string, error)
 	}
 }
 
-func loadConfig(dir string) (*configfile.ConfigFile, error) {
+func loadConfig(dir string) (cfg *configfile.ConfigFile, err error) {
 	if dir == "" {
 		dir = config.Dir()
 	}
 
-	cfg, err := config.Load(dir)
-	if err != nil {
-		return nil, err
+	dockerConfigJSON := filepath.Join(dir, config.ConfigFileName)
+	cfg = configfile.New(dockerConfigJSON)
+
+	switch _, err := os.Stat(dockerConfigJSON); {
+	case err == nil:
```

Comment:
Does it make sense to always attempt to read both files and then merge them?

Reply:
@jpower432 Any thoughts on this?

Reply:
There is no function in the `config` library that I saw capable of merging, but I could write a helper function to merge the two configs. I definitely think that it would cover more use cases, but we would have to have a preferred option when merging. Assuming we would use the `docker` config as the preferred option?

Reply:
Yeah favoring the docker config option makes sense so we avoid breaking people that already have both and have different credentials for the same registry in the podman auth file.

Reply:
This is a nit, btw. If its hard to do, I don't consider it a must-have.

Reply:
Understood. I don't believe it would be overly complicated, but I would like to tackle it in a follow-on PR if that works.